build.gradle

```diff
@@ -222,6 +215,15 @@ applicationDistribution.from("LICENSE.txt") {
     into ""
 }
 
+resolve {
```

Comment:
Mind adding a comment?  I had to look up what plugin this was for.  I suggest:

`Resolves project openAPI specification and saves the result in JSON`

Reply:
great point. updated.